tests/end2end/test_single_operator_with_cntk_backend.py

```diff
@@ -162,12 +162,15 @@ def test_dense(self):
         x = _create_tensor(N, C)
 
         input = Input(shape=(C,))
-        result = Dense(D)(input)
-        keras_model = Model(input=input, output=result)
+        hidden = Dense(D, activation='relu')(input)
+        result = Dropout(0.2)(hidden)
+
+        keras_model = Model(inputs=input, outputs=result)
         keras_model.compile(optimizer='adagrad', loss='mse')
 
-        coreml_model = coremltools.converters.keras.convert(keras_model)
-        onnx_model = onnxmltools.convert_coreml(coreml_model)
+        # coremltool cannot convert this kind of model.
+        # coreml_model = coremltools.converters.keras.convert(keras_model)
```

Comment:
If so, please create another tests for `Dropout`. If Core ML is not able to handle it, you can call `_test_one_to_one_operator_core_keras`.

Reply:
added a new one.